[JIT] Factor out aliases to separate test

eellison opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#38746 [JIT] Factor out aliases to separate test**
* #38735 [JIT] Normalize op aliases
* #38734 [JIT] Rename canonicalize ops

Factors out testing of op alias normalization so that there is a registry used for tests.

Differential Revision: [D21673107](https://our.internmc.facebook.com/intern/diff/D21673107)

Commit:
Factor out aliases to separate test

[ghstack-poisoned]

Commit:
Update on "[JIT] Factor out aliases to separate test"


Factors out testing of op alias normalization so that there is a registry used for tests.

[ghstack-poisoned]

Commit:
Update on "[JIT] Factor out aliases to separate test"


Factors out testing of op alias normalization so that there is a registry used for tests.

[ghstack-poisoned]

jamesr66a reviewed (approved):
![image](https://user-images.githubusercontent.com/4685384/82496519-c1335380-9aa1-11ea-823c-0d218d8d183f.png)

Commit:
Update on "[JIT] Factor out aliases to separate test"


Factors out testing of op alias normalization so that there is a registry used for tests.

[ghstack-poisoned]
